### PR TITLE
add portuguese 🇵🇹

### PR DESCRIPTION
**Pull request by joshreisner**
from https://github.com/code4recovery/12-step-meeting-list/discussions/1451

### CODE REVIEW

File: src/types.php [hunk 1]
```diff
@@ -368,16 +404,18 @@
...
   ),
    'L' => 
   (object) array(
      'en' => 'Lesbian',
-     'es' => 'Lesbianas',
+     'es' => 'Lesbiana',
      'fr' => 'Lesbienne',
      'ja' => 'レズビアン',
      'nl' => 'Lesbisch',
+     'pt' => 'Lésbica',
```
Comment (joshreisner) [inline src/types.php:418]: this is the one change i made - we don't want this translated as "Lesbians" -- same for Spanish

**Review by anchovie91471 — APPROVED** [review]
Looks good to me.

File: README.md [hunk 2]
```diff
@@ -346,107 +346,107 @@ The codes below should be considered 'reserved.' In your implementation, it's ok
...
-|`D-HOH`|Deaf / Hard of Hearing|Sordo / Duro de Oído|Sourd / Malentendant|聴覚障害者|Doof / Hardhoren|Nepočujúci / Nedoslýchaví|Döv / Hörselskadad|
-|`LO-I`|Loners / Isolationists|Solitarios / Aislacionistas|Solitaires / Isolationnistes|孤独 / 孤立主義者|Eenlingen / Isolationisten|Samotári / Izolacionisti|Ensamvargar / Isolationister|
-|`POA`|Proof of Attendance|Prueba de Asistencia|Preuve de Présence|出席証明|Bewijs van Aanwezigheid|Doklad o účasti|Närvarobevis|
-|`QSL`|Quebec Sign Language|Lengua de Señas de Quebec|Langue des Signes Québécoise|ケベック手話|Quebec -gebarentaal|Quebecký posunkový jazyk|Quebecskt Teckenspråk|
-|`RSL`|Russian Sign Language|Lengua de Señas Rusa|Langue des Signes Russe|ロシア手話|Russische gebarentaal|Ruský posunkový jazyk|Ryskt Teckenspråk|
+|Code|English|Español|Français|日本語|Nederlands|Português|Slovenčina|Svenska|
+|---|---|---|---|---|---|---|---|---|
+|`BV-I`|Blind / Visually Impaired|Ciego / Discapacidad Visual|Aveugle / Malvoyant|視覚障害者|Blind / Visueel gehandicapt|-|Nevidiaci / Zrakovo postihnutí|Blind / Synskadad|
```
[the remark this answers is not in the document]
Comment (joshreisner) [reply README.md:434]: oh yeah i didn't ask him to translate these - i think it's fine for now to proceed without

pretty cool behavior that it didn't die on these and just put a `-`

[the remark this answers is not in the document]
Comment (joshreisner) [reply README.md:434]: for completeness i pushed a change with these as well

[the remark this answers is not in the document]
Comment (anchovie91471) [reply README.md]: Cool beans!

**Pull request merged**